src/graphic/wordwrap.cc

```diff
@@ -38,10 +38,9 @@
 
 namespace {
 inline std::string as_editorfont(const std::string& text, int ptsize, const RGBColor& clr) {
-	// UI Text is always bold due to historic reasons
 	return format(
-	   "<rt keep_spaces=1><p><font face=sans size=%i bold=1 shadow=1 color=%s>%s</font></p></rt>",
-	   ptsize, clr.hex_value(), richtext_escape(text));
+	   "<rt keep_spaces=1><p><font face=sans size=%i shadow=1 color=%s>%s</font></p></rt>", ptsize,
+	   clr.hex_value(), richtext_escape(text));
```

Comment:
Is this changed for a specific reason or just to unify the look?

Reply:
We have to unify the look in any case since we now use WordWrap for both variants of edit boxes (previously single line edit box did not use word wrap, all rendering code was duplicated). IMHO bold text is a bit harder to read than regular text so I chose to go with the singleline's design here and drop the bold tag.